moduleframework/helpers/openshift_helper.py

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+#
+# This Modularity Testing Framework helps you to write tests for modules
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#
+
+import json
+import os
+from moduleframework import common
+from moduleframework.common import CommonFunctions
+from moduleframework.mtfexceptions import ContainerExc, ConfigExc
+
+
+class OpenShiftHelper(CommonFunctions):
```

Comment:
So how do I actually use this?

Reply:
It is used from commandline `sudo MODULE=openshift mtf` which executes this class.

Reply:
sweet! include it in docs please